dataland-frontend/src/components/forms/parts/kpiSelection/DataPointForm.vue

```diff
@@ -147,6 +151,10 @@ export default defineComponent({
       type: String,
       default: "percent",
     },
+    reportsName: {
```

Comment:
This seems unused

Reply:
Removed, could be introduced by the upload documents story, there were some changes merged with cherrypicking